packages/SwingSet/src/kernel/virtualObjectManager.js

```diff
@@ -389,8 +388,8 @@ export function makeVirtualObjectManager(
         } else {
           // exported non-virtual object: Remotable
           const remotable = requiredValForSlot(vref);
-          /** @type {number} */
-          const oldRefCount = (remotableRefCounts.get(remotable));
+          const oldRefCount = remotableRefCounts.get(remotable);
+          assert(oldRefCount !== undefined, `unknown remotable`);
```

Comment:
@FUDCo is this assertion accurate?

Reply:
I think it is irrelevant, as the code in question is different in the yet-to-be-landed PR for stores and collections.  For reference, the new code will be: ```
          // exported non-virtual object: Remotable
          const remotable = requiredValForSlot(vref);
          if (remotableRefCounts.has(remotable)) {
            /** @type {number} */
            const oldRefCount = (remotableRefCounts.get(remotable));
            remotableRefCounts.set(remotable, oldRefCount + 1);
          } else {
            remotableRefCounts.set(remotable, 1);
          }
```
where the `undefined` case is both permitted and handled, with a call to `has` differentiating which case we're in.

Reply:
Upon further examination, I can't determine what code you're proposing to fiddle with here, since it matches several fragments that are now in a different file.  But the assert is wrong in any case.

Reply:
To be more clear: don't do this.  This code has a different evolutionary path to take and this change will just mess that up.

Reply:
I'm confused because it should be strictly a subset of the `oldRefCount > 0` below. Aka `undefined` will never be `> 0`.

Reply:
I need to include some change, as as-is the code doesn't lint (type checks fail)

Reply:
`undefined` is handle as a separate branch of an `if`.  Stop trying to make sense of this; this code is changing.